src/app/qgsdecorationlayoutextentdialog.cpp

```diff
@@ -42,6 +42,8 @@ QgsDecorationLayoutExtentDialog::QgsDecorationLayoutExtentDialog( QgsDecorationL
 
   updateGuiElements();
   connect( buttonBox->button( QDialogButtonBox::Apply ), &QAbstractButton::clicked, this, &QgsDecorationLayoutExtentDialog::apply );
+  connect( buttonBox, &QDialogButtonBox::helpRequested, this, &QgsDecorationLayoutExtentDialog::showHelp );
+// Pas écrit pareil ci-dessus
```

Comment:
The first one directly connects to the button, the second one to a helper signal on the box. The second one only works with what's listed here under signals: http://doc.qt.io/qt-5/qdialogbuttonbox.html . I'm not aware of anythings that makes one of them preferable.